unreleased.rst

```diff
@@ -12,3 +12,4 @@ HEAD — ongoing
 * [DAML Docs] suppress instance documentation when `--data-only` mode is requested
 + [DAML-LF] add CAST_NUMERIC and SHIFT_NUMERIC in DAML-LF 1.dev
 + [Sandbox] Dramatically increased performance of the ActiveContractService by only loading the contracts that the parties in the transaction filter are allowed to see.
++ [DAML-LF] change signature of MUL_NUMERIC and DIV_NUMERIC
```

Comment:
I'm not sure we should track changes like this in the _user facing_ changelog. The numeric type is still very much under development and adding changelog entries for this adds a lot of noise and might give others the impression we're doing tons of breaking changes.
What do you think @gerolf-da?

Reply:
@hurryabit: sounds reasonable. Maybe for daml-lf 1.dev it makes sense to keep an internal changelog? this way we keep track of all the changes without adding it to the SDK changelog.

Reply:
@hurryabit You'r right. 
@georg-da We have already a change log for dev in the DAML-LF specification. It should be enough to track change here.

Reply:
I propose that we promote the changes to DAML-LF 1.dev from the DAML-LF spec changelog to the proper changelog when we freeze a version of DAML-LF.